chsdi/models/vector/stopo.py

```diff
@@ -1878,6 +1878,26 @@ class GeologieRohstoffeZementAbbauVerarbeitung(Base, Vector):
 register('ch.swisstopo.geologie-rohstoffe-zement_abbau_verarbeitung', GeologieRohstoffeZementAbbauVerarbeitung)
 
 
+class GeologieRohstoffeZiegelVerarbeitung(Base, Vector):
+    __tablename__ = 'rohstoffe_ziegel_verarbeitung'
+    __table_args__ = ({'schema': 'geol', 'autoload': False})
+    __bodId__ = 'ch.swisstopo.geologie-rohstoffe-ziegel_verarbeitung'
+    __template__ = 'templates/htmlpopup/rohstoffe_ziegel_verarbeitung.mako'
+    __queryable_attributes__ = ['obname', 'pckind']
+    __label__ = 'obname'
+    id = Column('bgdi_id', Integer, primary_key=True)
```

Comment:
there is a unique and not null attribute ``id_objekt`` in the table, if this attribute is provided by the data owner please use this attribute as primary key instead, but please verifiy the source of this attribute first. if it is only a meaningless counter, we keep bgdi_id.

Reply:
For each Rohstoffe layer, the primary key is put on the bgdi_id, I think we can leave like that here too

Reply:
yes you're right